js_modules/dagit/packages/core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -22,16 +22,14 @@ import {displayNameForAssetKey} from '../asset-graph/Utils';
 import {PartitionHealthSummary} from '../assets/PartitionHealthSummary';
 import {AssetKey} from '../assets/types';
 import {
+  ConfigPartitionSelectionQueryQuery,
```

Comment:
any reason these were renamed?  (e.g. `QueryQuery`)

Reply:
The new codegen tool appends `Query`, `Mutation`, `Fragment` etc. to the generated type names. It might be something we can turn off, but for now it's also a useful indicator that it's a new type versus an old one.